src/EvoSC.CLI/CliHandler.cs

```diff
@@ -0,0 +1,79 @@
+﻿using System.CommandLine;
+using System.Reflection;
+using EvoSC.CLI.Attributes;
+using EvoSC.CLI.Exceptions;
+using EvoSC.CLI.Interfaces;
+
+namespace EvoSC.CLI;
+
+public class CliHandler
+{
+    private RootCommand _rootCommand;
+    private readonly string[] _args;
+    
+    public CliHandler(string[] args)
+    {
+        _args = args;
+        _rootCommand = new RootCommand("EvoSC TrackMania Server Controller.");
```

Comment:
Should be `EvoSC# Trackmania Server Controller`

Reply:
fixed.